utils/releaser/src/ReleaseWorker/Release/CreateAndPushGitTagReleaseWorker.php

```diff
@@ -30,14 +30,13 @@ public function work(
         Version $version,
         string $initialBranchName = AbstractShopsysReleaseWorker::MAIN_BRANCH_NAME,
     ): void {
-        $versionString = $version->getOriginalString();
-        $this->processRunner->run('git tag ' . $versionString);
+        $tag = $version->getOriginalString();
+        $this->processRunner->run('git tag ' . $tag);
         $this->symfonyStyle->note(
-            sprintf('You need to push tag manually using "git push origin %s" command.', $versionString),
+            sprintf('You need to push tag manually using "git push origin %s --tags" command.', $initialBranchName),
```

Comment:
Is this necessary? I thought the push with the name of tag will push this tag, or?

Reply:
changed to 2 steps as we discussed :wink: